doc/CONFIGURATION.md

```diff
@@ -104,6 +104,22 @@ This may be disabled and scale 1 will be used, unless a `SCALE` has been specifi
 AUTO_SCALE: false
 ```
 
+### AUTO_SCALE_MIN
+
+Set minimum value for auto scaling. The default is 1.0.
+
+```yaml
+AUTO_SCALE_MIN: 1.0
+```
+
+### AUTO_SCALE_MAX
+
+Set maximum value for auto scaling. Values below 1.0 are ignored.
```

Comment:
Can we just accept values below 1.0? There might be a use case...

Reply:
Yeah, I changed that.